README.md

```diff
@@ -52,42 +52,291 @@ With most carrying a footer containing:
+| `players[].avatar`             | object  | The avatar object of the player.                                                                                                |
+| `players[].avatar.url`         | string  | The avatar image URL (40px * 20px).                                                                                             |
+| `players[].avatar.tooltip`     | string  | The avatar tooltip text.                                                                                                        |
+
+
+
+
+
+
+## Social
+
+**TODO*** (Downlord:) Would it make sense to merge this into the login response?
+**TODO*** (Downlord:) `channels` and `autojoin` seem redundant.
+
+#### Push
+
+    {
+      "power": integer,
+      "autojoin": [
+        string
+      ],
+      "foes": [
+        integer
+      ],
+      "command": string,
+      "channels": [
+        string
+      ],
+      "friends": [
+        integer
+      ]
+    }
+
+| Parameter    | Value  | Description                                        |
+|--------------|--------|----------------------------------------------------|
+| `power`      | string | The administrative power given to the user.        |
+| `autojoin[]` | list   | A list of chat channels (`string`) to join.        |
+| `foes`       | list   | A list of player IDs (`integer`) that are foes.    |
+| `friends`    | list   | A list of player IDs (`integer`) that are friends. |
+| `channels[]` | list   | A list of chat channels (`string`) to join.        |
+
+
+
+
+
+
+## Hosting game
+
+**TODO*** (Downlord:) Would it make sense to merge this into the login response?
+**TODO*** (Downlord:) `channels` and `autojoin` seem redundant.
+
+#### Request
+
+    {
+      "command": "game_host",
+      "gameport": integer,
+      "mapname": string,
+      "title": string,
+      "mod": string,
+      "access": string,
+      "visibility": string
+    }
+
+| Parameter    | Type    | Description                                                                          |
+|--------------|---------|--------------------------------------------------------------------------------------|
+| `gameport`   | map     | The technical name of the map.                                                       |
+| `mapname`    | string  | The technical name of the map.                                                       |
+| `title`      | integer | The title of the game.                                                               |
+| `mod`        | integer | The name of the active game modification.                                            |
+| `access`     | integer | The accessability of the game, one of `["public", "private"]`. TODO is this correct? |
+| `visibility` | string  | The visibility of the game, one of `["public", "private"]`. TODO is this correct?    |
+
+#### Response
+
+**TODO*** (Downlord:) `id` instead of `uid` (and/or: use real UUIDs)
```

Comment:
Games should keep using atomically increasing unsigned integer identifiers, there's no reason to use UUID's for them.